packages/rest/src/router/routing-table.ts

```diff
@@ -143,15 +135,14 @@ export class RoutingTable {
       return found;
     }
 
-    // this._staticAssetsRoute will be set only if app.static() was called
```

Comment:
Please note this comment is no longer true, RestServer is always creating a new instance of `StaticAssetsRoute`/`ExternalRoutes`. I am not sure if that's a good thing and an intentional behavior. Either way, fixing it is out of scope of this pull request.